.env.development

```diff
@@ -1,5 +1,6 @@
-VITE_AUTH_URL=http://localhost:8080/v1/auth/authorize
-VITE_TOKEN_URL=http://localhost:8080/v1/auth/token
-VITE_CLIENT_ID=dashboard
-VITE_REDIRECT_URI=http://localhost:5173/callback
-VITE_SCOPE=profile email
+PUBLIC_AUTH_URL=http://localhost:8080/v1/auth/authorize
```

Comment:
can we prefix them with `OAUTH` when we are changing them anyway?

Reply:
Like now?

Reply:
I thought you need the PUBLIC prefix, otherwise it won't work? I meant like `PUBLIC_OAUTH....`